app/services/source-filters.ts

```diff
@@ -121,11 +121,7 @@ export class SourceFiltersService extends Service {
       }
 
       /* Video filters can be applied to video sources. */
-      if (source.video && filterType.video) {
-        return true;
-      }
-
-      return false;
+      return source.video && filterType.video;
```

Comment:
Same thing.  Why break the pattern of the function?  Why not combine this entire function into 1 big conditional?  (please don't do that)